sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaProperties.cs

```diff
@@ -35,5 +36,10 @@ internal SchemaProperties(SchemaFormat format, string schemaId, string groupName
         /// The name of the schema.
         /// </summary>
         public string Name { get; }
+
+        /// <summary>
+        /// The version of the schema.
+        /// </summary>
+        public int? Version { get; }
```

Comment:
Hmm.. Why is this nullable? I guess it is based on the swagger not marking it as required..

Reply:
Yeah I was having trouble passing it through from the response since it's nullable there, it didn't seem to make sense to give it a default value like 0 or 1 or something, so I left it as nullable

Reply:
We should be able to customize this, but I wonder if we should ask Nick from the service team if it is correct that these properties are not listed as required.

Reply:
Turns out you can't mark response headers as required in swagger. We can just make this non-nullable though.